system/services/assetManager/AssetQueueService.cfc

```diff
@@ -107,12 +115,14 @@ component implements="preside.system.services.assetManager.IAssetQueue" {
 		transaction {
 			var takenByOtherProcess = false;
 			var queueDao            = $getPresideObject( "asset_generation_queue" );
+			var assetTenantField    = Trim( _getTenancyService().getObjectTenant( "asset" ) );
```

Comment:
This is not necessarily the field, rather the _tenant_. It _usually_ is the FK field, but doesn't have to be. Use `getTenantFkForObject( "asset" )` instead. (throughout this PR)